src/app/(tools)/coverLetter/page.tsx

```diff
@@ -0,0 +1,285 @@
+"use client";
+
```

Comment:
too big file, a lot is happening here, that we can move into separate files and keep it clean